images/azure/rocky-9.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-build_name: "rocky-93-az"
+build_name: "rocky-9-az"
 packer_builder_type: "azure"
 python_path: ""
 packer:
```

Comment:
should image_version be latest? or we want to leave it as is

Reply:
🤦 yep, fixed.